src/k8s-engine/lima.ts

```diff
@@ -147,12 +155,19 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
   /** The port the Kubernetes server _should_ listen on */
   #desiredPort = 6443;
 
+  /** Currently either containerd or moby, changing requires a full restart */
+  #currentContainerEngine = ContainerEngine.NONE;
+
+  protected changedContainerEngine = false;
```

Comment:
When does this get set? When does it get cleared? JSDoc would be nice.

Reply:
added